gbpservice/nfp/configurator/lib/fw_constants.py

```diff
@@ -13,9 +13,11 @@
 DRIVERS_DIR = 'gbpservice.nfp.configurator.drivers.firewall'
 SERVICE_TYPE = 'firewall'
 VYOS = 'vyos'
+ASAV = 'asav'
 NEUTRON = 'neutron'
 
 CONFIGURATION_SERVER_PORT = '8888'
+ASAV_CONFIGURATION_SERVER_PORT = '443'
```

Comment:
Move all the ASAV and VYOS related constants to different files and keep then in drivers directory itself. Keep all the common constants related to firewall here.